hydra/test_utils/launcher_common_tests.py

```diff
@@ -538,12 +555,16 @@ def test_to_absolute_path_multirun(
             overrides=overrides,
             prints="hydra.utils.to_absolute_path('/foo/bar')",
             expected_outputs=path,
+            post_run_callback=self.post_run_callback(),
         )
-        outputs = [str(tmpdir / "foo/bar"), str(tmpdir / expected_dir)]
+        working_dir = expected_working_dir if expected_working_dir else tmpdir
```

Comment:
you have an allocated LOCAL temp directory. can you download your files into that instead of juggling between the two output directories?

Reply:
I think  the naming of the variable is confusing. I've renamed it as previously mentioned. Pls let me know if it still doesnt make sense.

Reply:
This test is testing that the hydra.utils.to_absolute_path() is working correctly.
It's intended to be executed and tested in the job.

Is this change changing the expected output to that of the remote run?

Reply:
yes, changing this to the remote working dir of the test application.